samples/bot-formatting-cards/csharp/README.md

```diff
@@ -125,6 +125,47 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
             }
             ]
         ```
+   
+   **Note: In Adaptive Card what we are defining (User details) should be exist in the same tenant where you are testing the app (teams login) etc...**
+  - Locate the Basic info section on the user's Profile page. The Object ID that is displayed is the user's unique object ID.
+    - Navigate to samples\bot-formatting-cards\csharp\BotFormattingCards\Resources\adaptivePeoplePersonaCardIcon.json
+      1) On line 16, replace {{User-Object-ID}}  
+      2) On line 17, replace {{User-Display-Name}}
+      3) On line 18, replace {{User-Principal-Name}}
+
+        - E.g. 
+        ```
+        "properties": {
+        "id": "87d349ed-xxxx-434a-9e14-xxxx",
+        "displayName": "Joe Smith",
+        "userPrincipalName": "Joe Smith@xxxx.com"
+      }
+        ```
+    - Navigate to samples\bot-formatting-cards\csharp\BotFormattingCards\Resources\adaptivePeoplePersonaCardSetIcon.json
+      1) On line 18, replace {{User-Object-ID}}  
+      2) On line 19, replace {{User-Display-Name}}
+      3) On line 20, replace {{User-Principal-Name}}
+      4) On line 23, replace {{User-Object-ID}}  
+      5) On line 24, replace {{User-Display-Name}}
+      6) On line 25, replace {{User-Principal-Name}}
+      
+        - E.g. 
+        ```
+      "properties": {
+        "users": [
+          {
+            "id": "87d349ed-xxxx-434a-9e14-xxxx",
+            "displayName": "Joe Smith",
+            "userPrincipalName": "Joe Smith@xxxx.com"
+          },
+          {
+            "id": "87d349ed-xxxx-434a-9e14-xxxx",
+            "displayName": "Joe Smith",
```

Comment:
Please don't put same name and details for both the entries.

Reply:
Fixed